actors/common/src/main/java/org/sunbird/learner/actors/otp/OTPActor.java

```diff
@@ -122,11 +120,35 @@ private void verifyOTP(Request request) {
       ProjectCommonException.throwClientErrorException(ResponseCode.errorInvalidOTP);
     }
 
+    if (otpInRequest.equals(otpInDB)) {
+      otpService.deleteOtp(type, key);
+    } else {
+      handleMismachOtp(type, key, otpDetails);
+    }
     Response response = new Response();
     response.put(JsonKey.RESPONSE, JsonKey.SUCCESS);
     sender().tell(response, self());
   }
 
+  private void handleMismachOtp(String type, String key, Map<String, Object> otpDetails) {

```

Comment:
handleMismachOtp typo to handleMismatchOtp

Reply:
done